docs/projects/ad7616_sdz/index.rst

```diff
@@ -0,0 +1,428 @@
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Product datasheets: :part:`AD7616`
+
+-  `UG-1012, Evaluation Board User Guide <https://www.analog.com/media/en/technical-documentation/user-guides/EVAL-AD7616SDZ-7616-PSDZ-UG-1012.pdf>`__
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`ad7616_sdz HDL project <main:projects/ad7616_sdz>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AD_EDGE_DETECT
+     - :git-hdl:`library/common/ad_edge_detect.v <main:library/common/ad_edge_detect.v>`
+   * - AXI_AD7616
+	 - :git-hdl:`library/axi_ad7616 <main:library/axi_ad7616>` *
+	 - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_ad7616>`
+   * - AXI_CLKGEN
+     - :git-hdl:`library/axi_dmac <main:library/axi_clkgen>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_clkgen>`
+   * - AXI_DMAC
+     - :git-hdl:`library/axi_dmac <master:library/axi_dmac>`
+     - :ref:`here <axi_dmac>`
+   * - AXI_HDMI_TX
+     - :git-hdl:`library/axi_hdmi_tx <main:library/axi_hdmi_tx>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_hdmi_tx>`
+   * - AXI_I2S_ADI
+     - :git-hdl:`library/axi_i2s_adi <main:library/axi_i2s_adi>`
+   * - AXI_PWM_GEN
+     - :git-hdl:`library/axi_pwm_gen <main:library/axi_pwm_gen>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_pwm_gen>`
+   * - AXI_SPDIF_TX
+     - :git-hdl:`library/axi_spdif_tx <main:library/axi_spdif_tx>`
+   * - AXI_SPI_ENGINE
+     - :git-hdl:`library/spi_engine/axi_spi_engine <main:library/spi_engine/axi_spi_engine>`  **
+     - :ref:`here <spi_engine axi>`
+   * - AXI_SYSID
+     - :git-hdl:`library/axi_sysid <main:library/axi_sysid>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_sysid>`
+   * - AXI_SPI_ENGINE
+     - :git-hdl:`library/spi_engine/axi_spi_engine <main:library/spi_engine/axi_spi_engine>`  **
+     - :ref:`here <spi_engine axi>`
+   * - SPI_ENGINE_EXECUTION
+     - :git-hdl:`library/spi_engine/spi_engine_execution <main:library/spi_engine/spi_engine_execution>` **
+     - :ref:`here <spi_engine execution>`
+   * - SPI_ENGINE_INTERCONNECT
+     - :git-hdl:`library/spi_engine/spi_engine_interconnect <main:library/spi_engine/spi_engine_interconnect>` **
+     - :ref:`here <spi_engine interconnect>`
+   * - SPI_ENGINE_OFFLOAD
+     - :git-hdl:`library/spi_engine/spi_engine_offload <main:library/spi_engine/spi_engine_offload>` **
+     - :ref:`here <spi_engine offload>`
+   * - SYNC_BITS
+     - :git-hdl:`library/util_cdc/sync_bits.v <main:library/util_cdc/sync_bits.v>`
+   * - SYSID_ROM
+     - :git-hdl:`library/sysid_rom <main:library/sysid_rom>`
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for SER_PAR_N=0 (parallel interface)
+   -   ``**`` instantiated only for SER_PAR_N=1 (serial interface)
+
+-  :ref:`ADI HDL User guide <user_guide>`
```

Comment:
Lines 416-418 should be removed, as there are already references to these in here

Reply:
Done.